packages/manageiq-ansible-venv/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM registry.access.redhat.com/ubi8/ubi:8.2
+
+ARG ARCH=x86_64
+
+ENV NAME=manageiq-ansible-venv \
+    VERSION=1.0.0 \
+    VENV_ROOT=/var/lib/manageiq \
+    RPM_SPEC=manageiq-ansible-venv.spec
+
+RUN dnf -y --disableplugin=subscription-manager install \
+      http://mirror.centos.org/centos/8.2.2004/BaseOS/${ARCH}/os/Packages/centos-repos-8.2-2.2004.0.1.el8.${ARCH}.rpm \
+      http://mirror.centos.org/centos/8.2.2004/BaseOS/${ARCH}/os/Packages/centos-gpg-keys-8.2-2.2004.0.1.el8.noarch.rpm \
```

Comment:
Can we identify which packages need these yum repos, so that if we remove them in the future we know to also remove the repo?

Reply:
Updated